apisix/http/route.lua

```diff
@@ -0,0 +1,42 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local expr = require("resty.expr.v1")
+local plugin_checker = require("apisix.plugin").plugin_checker
+
+
+local _M = {}
+
+
+-- additional check for synced route configuration, run after schema check
```

Comment:
If the title is like this：`additional "vars" rule check for synced route configuration, run after schema check`,  I think it would be better, right?

Reply:
@Firstsawyou 
There are other checks here (like plugin checker), so I don't think adding `"vars"` is a good idea.

Reply:
ok.